packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3844,10 +3868,10 @@ export default class Meeting extends StatelessWebexPlugin {
         .handleClientRequest(this, true)
         .then(() => {
           MeetingUtil.handleAudioLogging(this.mediaProperties.audioTrack);
-          Metrics.postEvent({
-            event: eventType.MUTED,
-            meeting: this,
-            data: {trigger: trigger.USER_INTERACTION, mediaType: MetricsMediaType.AUDIO},
+          NewMetrics.submitClientEvent({
+            name: 'client.muted',
+            payload: {trigger: 'user-interaction', mediaType: 'audio'},
+            options: {meetingId: this.id},
```

Comment:
just a thought: meetingId for all these events seems like something that's mandatory, so not sure if it should be in "options" - options sound like things that are "optional". Shouldn't meeting id be in payload.identifiers?

Reply:
The `payload` object contains "true" payload for the event, and by "true" I mean it follows the expected type/structure for the client event. It is not all of the payload tho, some of it is computed inside the new metrics class and we merge them together to form the final payload. The reason why meetingId is needed here is in order to compute the extra payload inside the class(locus related identifiers, correlationId, etc).

There will be events which won't require meetingId (as they will happen before the meeting exists), which is why it's optional.